example/server/README.md

```diff
@@ -0,0 +1,5 @@
+## To run the example auth sever
+
```

Comment:
Would be great to describe what the auth server does in this README.md and why its important to run it.